src/_DataStructures_/Trees/BinarySearchTree/bst-deletion.test.js

```diff
@@ -0,0 +1,37 @@
+const BinarySearchTree = require('./index');
+const heightOfBST = require('./height-of-bst/index');
+
+describe('Binary search tree traversals', () => {
+  let bst;
+
+  describe('Creates BST', () => {
+    it('should create BST', () => {
+      // Creates BST
+      bst = new BinarySearchTree(6);
+      bst.add(4);
+      bst.add(9);
+      bst.add(2);
+      bst.add(5);
+      bst.add(8);
+      bst.add(12);
+    });
+  });
```

Comment:
@balajipachai how you are checking that the tree is created the way you wanted to create it? You have only added nodes, haven't verified that the nodes are added properly to the tee. 

For that, you have to verify any of the traversal output.